libs/remix-ui/run-tab/src/lib/components/contractGUI.tsx

```diff
@@ -549,25 +557,52 @@ export function ContractGUI (props: ContractGUIProps) {
                   onChange={handleUseLastProxySelect}
                   checked={useLastProxy}
                 />
-                <label
-                  htmlFor="proxyAddress"
-                  data-id="contractGUIProxyAddressLabel"
-                  className="m-0 form-check-label custom-control-label udapp_checkboxAlign"
-                  title="Select this option to use the last deployed ERC1967 contract on the current network."
-                  style={{ fontSize: 12 }}
+                <CustomTooltip
+                  tooltipText="Select this option to use the last deployed ERC1967 contract on the current network."
+                  tooltipId="proxyAddressTooltip"
+                  placement="auto"
+                  tooltipClasses="text-wrap"
                 >
-                  Use last deployed ERC1967 contract
-                </label>
+                  <label
+                    htmlFor="proxyAddress"
+                    data-id="contractGUIProxyAddressLabel"
+                    className="m-0 form-check-label custom-control-label udapp_checkboxAlign"
+                    style={{ fontSize: 12 }}
+                  >
+                    Use last deployed ERC1967 contract
+                  </label>
+                </CustomTooltip>
               </div>
               {
                 !useLastProxy ?
                 <div className="mb-2">
-                  <label className='mt-2 text-left d-block'>Proxy Address: </label>
-                  <input style={{ height: 32 }} className="form-control udapp_input" data-id="ERC1967AddressInput" placeholder='proxy address' title='Enter previously deployed proxy address on the selected network' onChange={handleSetProxyAddress} onBlur={() => validateProxyAddress(proxyAddress) } />
-                  { proxyAddressError && <span className='text-lowercase' data-id="errorMsgProxyAddress" style={{ fontSize: '.8em' }}>{ proxyAddressError }</span> }
-                </div> :
-                <span className='text-capitalize' data-id="lastDeployedERC1967Address" style={{ fontSize: '.8em' }}>{ proxyAddress || proxyAddressError }</span>
-              }
+                  <label className="mt-2 text-left d-block">
+                    Proxy Address:{" "}
```

Comment:
I think this is an error in this line `{" "}`

Reply:
Yes This was a mistake with prettier. Fixed